readme.md

```diff
@@ -134,44 +136,62 @@ If you need to override the default values, provide the following options:
 
 ## Local authentication & authorization emulation
 
-When requesting `./auth/login/<provider>`, you will access a local authentication UI allowing you to set fake user information.
+The CLI allows you to mock and read authentication & authorization credentials.
+
+> **NOTE:** You must use port `4280` to access the authentication UI.
```

Comment:
While `4280` is the default port, users may provide a different port using the `--port` option.

Reply:
I'll update the note to say that.